### PR TITLE
FR + LU: Remove links

### PR DESCRIPTION
**Pull request by BellezaEmporium**
Removed links from "orange.fr" since the server was shut down
Changed M6's name to M6 International, since it's clearly written, and since it's not the national version.

### CODE REVIEW

**Comment by Dum4G** [comment]
Check lu.m3u


#EXTINF:-1 tvg-id="RTL9.lu" tvg-name="RTL9" tvg-country="LU;FR;MC;CH" tvg-language="French" tvg-logo="https://photos.abweb.com/shared_ressources/logos/logo-rtl9.png" group-title="",RTL9 (720p)
--
29 | http://cdn.webtv4.cdnfr.orange.fr/hs/HALO3/hls/live/Rtl9_live/index.m3u8

**Comment by LaneSh4d0w** [comment]
> Check lu.m3u
> 
> ## #EXTINF:-1 tvg-id="RTL9.lu" tvg-name="RTL9" tvg-country="LU;FR;MC;CH" tvg-language="French" tvg-logo="https://photos.abweb.com/shared_ressources/logos/logo-rtl9.png" group-title="",RTL9 (720p)
> 29 | http://cdn.webtv4.cdnfr.orange.fr/hs/HALO3/hls/live/Rtl9_live/index.m3u8

Nice find.

File: channels/fr.m3u [hunk 1]
```diff
@@ -237,8 +233,8 @@ https://dcunilive47-lh.akamaihd.net/i/dclive_1@739146/master.m3u8
 https://alchimie-humanity-1-fr.samsung.wurl.com/manifest/playlist.m3u8
 #EXTINF:-1 tvg-id="KTO.fr" tvg-name="KTO" tvg-country="FR;UK;PT;ES;DK;NL;BE;CH" tvg-language="French" tvg-logo="https://i.imgur.com/LvbXFUy.jpg" group-title="",KTO (404p)
 https://livehdkto-lh.akamaihd.net/i/LiveStream_1@178944/master.m3u8
-#EXTINF:-1 tvg-id="LEquipe.fr" tvg-name="L'Équipe" tvg-country="FR" tvg-language="French" tvg-logo="https://i.ibb.co/bvhBK1y/LEquipe-21.png" group-title="",L'Équipe (576p)
-http://cdn.webtv4.cdnfr.orange.fr/hs/HALO3/hls/equipe21-850463/hls/index.m3u8
+#EXTINF:-1 tvg-id="LEquipe.fr" tvg-name="L'Équipe" tvg-country="FR" tvg-language="French" tvg-logo="https://i.ibb.co/bvhBK1y/LEquipe-21.png" group-title="",L'Équipe [Geo-blocked] (576p)
+https://query-streamlink.lanesh4d0w.repl.co/iptv-query?streaming-ip=https://www.dailymotion.com/video/x2lefik
```
Comment (freearhey) [inline channels/fr.m3u:237]: query-streamlink.lanesh4d0w.repl.co is down

**Review by freearhey — CHANGES_REQUESTED** [review]

File: channels/fr.m3u (hunk 1 above)
Comment (LaneSh4d0w) [reply channels/fr.m3u:237]: All good now.

Comment (freearhey) [reply channels/fr.m3u:237]: For me, the error code just changed to 500.
image
Or should that be the case if the source is geo-blocked?

Comment (LaneSh4d0w) [reply channels/fr.m3u:237]: Well I think that it won't work since it's the IP of the server that does the request. Taking it off.

**Review by freearhey — APPROVED** [review]

**Pull request merged**